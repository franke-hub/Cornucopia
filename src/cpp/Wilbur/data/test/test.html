<HTML>
<HEAD><TITLE>Wilbur-Experimental/0.1 index.html</TITLE></HEAD>
<BODY>
This xxyyz is a test of HTML PARSER.
<H1 align="center">Home Page</H1>
<br><a href="http://www.google.com/">GOOGLE</a>
<br><a href="/giggle.html">GIGGLE (error)</a>
<br><a href="/.giggle.html">.GIGGLE (error)</a>
<br><a href="/input.html">Input form</a>
<br><a href="/shutdown.html">Shut down</a>
<hr>VID 2010/03/22 16:00 EDT
</BODY></HTML>
