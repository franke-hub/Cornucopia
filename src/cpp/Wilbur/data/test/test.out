      660434 a
         473 aa
         169 aaa
          62 aaaa
          22 aaaaa
           6 aaaaaa
           6 aaaaaaa
           4 aaaaaaaa
          14 aaaaaaaaa
           6 aaaaaaaaaa
           1 aaaaaaaaaaa
           1 aaaaaaaaaah
           2 aaaaab
           6 aaaaarg
          14 aaaab
           1 aaaabddeeehillmnprrrstttty
           2 aaab
           2 aaaeq
           1 aaagh
           1 aaah
           6 aaai
          41 aab
           4 aabaabaabaab
           2 aabbcc
           1 aabbccddeeff
           4 aac
           1 aacaridius
           4 aaccounts
           5 aachen
          80 aactionnames
          24 aacute
           1 aacutesmall
          21 aad
           4 aaddress
           4 aadimen
           1 aadvantage
           3 aaf
           2 aafa
           4 aaffff
           3 aah
           4 aahz
           1 aaiii
           1 aaiiieee
           1 aal
         252 aallgroups
          16 aam
          10 aan
           2 aand
           1 aanpa
           2 aanpassing
           7 aanswer
           2 aantt
           2 aao
          50 aaon
           6 aapple
           2 aar
           3 aardvark
         279 aargs
           3 aarguments
           2 aarhus
          29 aaron
           2 aarvold
          23 aas
           5 aastex
           1 aastha
           3 aat
           2 aaudioarray
           9 aaudiospinner
           3 aavsetmenuarray
           1 aawbvzx
         264 ab
          26 aba
           1 ababab
           6 ababcdefghij
           3 aback
           4 abacking
           2 abackslash
           6 abacus
          24 ain't
           1 ainavcawmemvaimh
           2 ainctl
           5 ainda
           4 ainsert
          31 ainsi
           4 ainsworth
           8 aint
